### DISCUSSION TITLE
Power Virtual Agents Kick-Off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#8] [Power Virtual Agents]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by mllorcag** [comment]
Thank you @FBoucher !!

I have a couple of questions. Regarding the topics covered:

1. Building chatbots and virtual agents for customer engagement.
2. Designing conversational flows and natural language understanding.
3. Integrating chatbots with other Power Platform components.
4. Monitoring and analyzing bot performance and user interactions.
5. Enhancing chatbots with AI capabilities.

When you mentioned "Building chatbots and virtual agents for customer engagement" do you mean "integration with Dynamics 365 -former CE-"? or is referred to just "how you can engage your customers thanks to PVA?".

The other question I have is related to delivery times. When is the deadline?

**Reply by fboucher** [reply]
Hi @mllorcag I mean users :)  (customer of the services app where you published the agent)

There is no official deadline. I understand it's summer and many are taking time off... That said it would be great to have something to show mid-September. Not a final version, but something. :)

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)

@mllorcag if you didn't get access ping me!
